site/en/guides/border-router/espressif-esp32.md

```diff
@@ -1,69 +1,73 @@
+Contributor: https://github.com/gjc13 https://github.com/zwx1995esp
 
 ESP Thread border router is a FreeRTOS-based implementation running on a combination of Espressif's Wi-Fi and 802.15.4 SoCs.
 
 Hardware requirements:
 
-* An ESP32 series Wi-Fi dev kit. (ESP32, ESP32-C3, ESP32-S2 or ESP32-S3)
-* An [ESP32-H2](https://www.espressif.com/en/news/ESP32_H2) dev kit for running RCP.
+Espressif provides an [ESP Border Router Board](https://www.espressif.com/en/news/Thread_Border_Router_Certification) which integrates the host SoC and the RCP into one module.
 
-## Hardware connection
-
-The ESP32-H2 RCP will be connected to the border router via UART.
+## Hardware platforms
 
 <figure>
-<a href="../../guides/images/otbr-esp-connection.jpg"><img src="../../guides/images/otbr-esp-connection.jpg" width="600" border="0" alt="ESP32-H2 UART" /></a><figcaption>ESP32-H2 UART connection</figcaption>
+<a href="../../guides/images/otbr-esp-br-board.png"><img src="../../guides/images/otbr-esp-br-board.png" width="600" border="0" alt="ESP Thread Border Router Board" /></a><figcaption>ESP Thread Border Router Board</figcaption>
 </figure>
 <br/>
 
-* Connect the ground PIN of ESP32 and ESP32-H2
-* Connect PIN 4 of ESP32 to PIN TX of ESP32-H2
-* Connect PIN 5 of ESP32 to PIN RX of ESP32-H2
-
-## Step 1: Set up ESP-IDF
-
-The code for running the ESP Thread border router is shipped with the Espressif development framework, ESP-IDF.
+## Step 1: Set up Repositories
 
-To set up the environment, please follow the [official installation guide](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#installation-step-by-step).
+To set up the environment, please follow the [official installation guide](https://docs.espressif.com/projects/esp-idf/en/stable/esp32s3/get-started/index.html#installation-step-by-step).
 
-## Step 2: Build and flash the RCP
+Clone the [esp-idf](https://github.com/espressif/esp-idf) and the [esp-thread-br](https://github.com/espressif/esp-thread-br) repository.
 
 ```
-$ cd ${IDF_PATH}/examples/openthread/ot_rcp
-$ idf.py --preview set-target esp32h2
-$ idf.py build flash
+$ git clone -b v5.1.1 --recursive https://github.com/espressif/esp-idf.git
+$ cd esp-idf
+$ ./install.sh
+$ . ./export.sh
+$ cd ..
+$ git clone --recursive https://github.com/espressif/esp-thread-br.git
 ```
 
-## Step 3: Configure the Thread and the Wi-Fi network
+## Step 2: Configure the Thread and the Wi-Fi network
 
 The border router will automatically join the Wi-Fi network and create a new Thread network if there is not one in its storage.
 The network parameters can be configured in the config menu:
 
 ```
-$ cd ${IDF_PATH}/examples/openthread/ot_br
+$ cd esp-thread-br/examples/basic_thread_border_router
 $ idf.py menuconfig
 ```
 
 The network configuration items are:
 
+* Enable automatic start mode in Thread Border Router **ESP Thread Border Router Example > Enable the automatic start mode in Thread Border Router.**
```

Comment:
Add a colon after "Router"

Enable automatic start mode in Thread Border Router: **ESP